feature/feature.go

```diff
@@ -44,6 +44,12 @@ const (
 	//
 	// alpha: v0.4
 	ClusterTopology featuregate.Feature = "ClusterTopology"
+
+	// KubeadmBootstrapFormatIgnition is a feature gate for the Ignition bootstrap format
+	// functionality.
+	//
+	// alpha: v1.0
```

Comment:
```suggestion
	// alpha: v1.1
```
nit (hopefully!!!! :))

Reply:
Fixed.